eml/components/login/LoginForm.js

```diff
@@ -22,141 +28,195 @@ import AsyncStorage from '@react-native-async-storage/async-storage';
  */
 export default function LoginForm() {
 
+    /* 
+        const [email, setEmail] = useState('');
+        const [password, setPassword] = useState('');
+        const [modalVisible, setModalVisible] = useState(false);
+        const [passwordAlert, setPasswordAlert] = useState('');
+        const [emailAlert, setEmailAlert] = useState('');
+    */
+
 	const navigation = useNavigation();
-	const [email, setEmail] = useState('');
-	const [password, setPassword] = useState('');
-	const [modalVisible, setModalVisible] = useState(false);
-	const [passwordAlert, setPasswordAlert] = useState('');
-	const [emailAlert, setEmailAlert] = useState('');
-	// State variable to track password visibility
-	const [showPassword, setShowPassword] = useState(false);
+
+    // state variable to track if user successfully submitted phone number
+    const [submittedPhoneNumber, setSubmittedPhoneNumber] = useState(false);
```

Comment:
```suggestion
    const [phoneSubmitted, setPhoneSubmitted] = useState(false);
```

Rename because it's a boolean, and not storing an actual submitted phone